README.md

```diff
@@ -508,6 +533,25 @@ const numberOfFinalizers = blockSummary.finalizationData.finalizers.length;
 ...
 ```
 
+Blocks before protocol version 4 have a different type than those from higher protocol versions.
+To determine the version, use `isBlockSummaryV1` and `isBlockSummaryV0`:
+
+```js
+...
+const blockSummary: BlockSummary = await client.getBlockSummary(blockHash);
+if (isBlockSummaryV0(blockSummary)) {
+    // blockSummary has version 0 structure 
```

Comment:
Maybe we should mention that this corresponds to protocol version <= 3, and likewise for version 1.